build.sbt

```diff
@@ -1,13 +1,19 @@
 // Dependencies are needed for Scala Steward to check if there are newer versions
 val akkaHttpVersion       = "10.2.2"
 val akkaVersion           = "2.6.10"
+$if(slick.truthy)$
 val slickVersion          = "3.3.3"
+$endif$
 val zioVersion            = "1.0.3"
 val zioLoggingVersion     = "0.5.4"
 val zioConfigVersion      = "1.0.0-RC31-1"
 val flywayVersion         = "7.3.2"
 val testContainersVersion = "0.38.8"
 val calibanVersion        = "0.9.4"
+$if(doobie.truthy)$
+val catsInteropVersion        = "2.2.0.1"
+val doobieVersion = "0.9.0"
```

Comment:
fix text alignment

Reply:
> Some formatting and style consistency issues. Otherwise looks good. Does Doobie has a connection pool? Or can use HickariDB ?

yes, doobie has HikariCP transactor.